Added method call `saveAssignments` to `\yii\rbac\PhpManager::removeItem`

rezident opened the pull request:
…tem`

Commit:
Added method call `saveAssignments` to `\yii\rbac\PhpManager::removeItem`

Commit:
Added method call `saveAssignments` to `\yii\rbac\PhpManager::removeItem`

SilverFire commented on framework/CHANGELOG.md:
```diff
@@ -53,6 +53,7 @@ Yii Framework 2 Change Log
 - Chg #9369: `Yii::$app->user->can()` now returns `false` instead of erroring in case `authManager` component is not configured (creocoder)
 - Chg #9411: `DetailView` now automatically sets container tag ID in case it's not specified (samdark)
 - Chg #9953: `TimestampBehavior::getValue()` changed to make value processing consistent with `AttributeBehavior::getValue()` (silverfire)
+- Bug #10101: Added method call `saveAssignments` to `\yii\rbac\PhpManager::removeItem` (rezident1307)
```
I think it's better to describe the fixed bug instead of changed line of code.
How about `Fixed assignments saving on role removing`?

SilverFire commented:
It happened because we do not have a proper test.
@rezident1307 could you add a test to [PhpManagerTest.php:143](https://github.com/yiisoft/yii2/blob/master/tests/framework/rbac/PhpManagerTest.php#L143)?

rezident1307 commented:
Ok. I will add a new commit later

samdark committed:
Merge pull request #10105 from githubjeka/patch-1

Update db-active-record.md [skip ci]

samdark committed:
Update db-active-record.md

PSR-2 4.1:
"Property names SHOULD NOT be prefixed with a single underscore to indicate protected or private visibility." (reverted from commit db7441f8668e364f208a0a58dab1437adddc57cd)

Commit:
Added test for fix assignments saving on role removing. Change CHANGELOG.md

Commit:
Merge branch 'fix' of github.com:rezident1307/yii2 into fix

rezident1307 commented:
Look a new pool request #10109
